src/assets/styles/base/_fonts.css

```diff
@@ -3,295 +3,295 @@
     font-family: 'Raleway';
     font-style: normal;
     font-weight: 400;
-    src: url('../fonts/raleway-v34-latin-regular.woff2') format('woff2');
+    src: url('/assets/fonts/raleway-v34-latin-regular.woff2') format('woff2');
```

Comment:
It's better to use a relative URL; if this isn't working we should figure out why.